torch/jit/__init__.py

```diff
@@ -643,7 +643,7 @@ def script(fn, optimize=True, _frames_up=0):
     # 2) Throwing everything away except for the graph 3) Creating a new
     # ScriptModule and dumping that graph in 4) Re-populating the schema
     # because it was lost doing the previous
-    mod.__getattr__('forward').forward_schema(ast, False)
+    mod.__getattr__('forward').forward_schema(ast, get_default_args(fn), False)
```

Comment:
Look for `_create_methods` in ScriptModule. Default arguments need to be added there as well.